client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -152,19 +152,28 @@ export const MetricsTab = () => {
 			<TimeDateChartControls onTimeRangeChange={ handleTimeRangeChange }></TimeDateChartControls>
 			<SiteMonitoringLineChart
 				title={ __( 'Requests per minute & average response time' ) }
+				tooltip={ __(
+					'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
+				) }
 				data={ formattedData as uPlot.AlignedData }
 			></SiteMonitoringLineChart>
 			<div className="site-monitoring__pie-charts">
 				<SiteMonitoringPieChart
-					title="Cache hit/miss"
+					title={ __( 'Cache hit/miss' ) }
+					tooltip={ __(
+						'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
+					) }
 					className="site-monitoring-cache-pie-chart"
 					data={ getFormattedDataForPieChart( cacheHitMissFormattedData, {
 						0: 'Cache miss',
 						1: 'Cache hit',
 					} ) }
 				></SiteMonitoringPieChart>
 				<SiteMonitoringPieChart
-					title="PHP vs. static content served"
+					title={ __( 'PHP vs. static content served' ) }
+					tooltip={ __(
+						'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```

Comment:
Percentages showing the ratio of dynamic versus static content. Dynamic content is the content generated using database information.